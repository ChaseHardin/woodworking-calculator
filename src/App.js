import React from 'react';
import logo from './logo.svg';
import './App.css';
import 'bootstrap/dist/css/bootstrap.min.css';
import { CutDetailsForm } from './components/cut-details-form/cut-details-form';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
      <CutDetailsForm />
    </div>
  );
}

export default App;
