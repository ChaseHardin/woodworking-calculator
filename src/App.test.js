import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

test('renders wrap application in container', () => {
  const tree = renderer.create(<App />).toJSON();

  expect(tree).toMatchSnapshot();
});