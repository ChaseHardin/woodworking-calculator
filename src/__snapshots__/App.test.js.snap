// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders wrap application in container 1`] = `
<div
  className="container"
>
  <div
    className="App"
  >
    <form
      className=""
      data-testid="cut-details-form"
    >
      <div
        className="form-group"
      >
        <label
          className="form-label form-label"
          data-testid="Length of Board"
        >
          Length of Board
        </label>
        <input
          className="form-control"
          placeholder="Length of board before cutting"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label form-label"
          data-testid="Boards"
        >
          Boards
        </label>
        <input
          className="form-control"
          placeholder="Number of boards after cutting"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label form-label"
          data-testid="Blade Width"
        >
          Blade Width
        </label>
        <input
          className="form-control"
          placeholder="Width of blade"
        />
      </div>
    </form>
  </div>
</div>
`;
